cmd/gen_release_notes/Makefile

```diff
@@ -5,7 +5,7 @@ BINARY := gen_release_notes
 
 VERSION = v0.4.0
 GIT_SHA = $(shell git rev-parse HEAD)
-LDFLAGS = -ldflags "-X main.gitSHA=$(GIT_SHA) -X main.version=$(VERSION) -X main.name=$(BINARY)"
+LDFLAGS = -ldflags "-X main.gitSHA=$(GIT_SHA) -X main.version=$(VERSION) -X main.name=$(BINARY) -linkmode external -extldflags '-static -Wl,--fatal-warnings'"
```

Comment:
Like I mentioned before, This flag triggered a build warning and then it generated a binary that I was not able to execute.

```
[werner@gtm-lap-024 ~/code/ecm-distro-tools/cmd/gen-release-notes]$ rm -f bin/gen-release-notes && make                                          
CGO_ENABLED=1 go build -tags "netgo osusergo no_stage static_build" -v -ldflags "-X main.gitSHA=03b906ee378cc1d06778121117a39415c8b82bb7 -X main.version=v0.4.0 -X main.name=gen-release-notes -extldflags '-static' -linkmode external" -o bin/gen-release-notes
# github.com/rancher/ecm-distro-tools/cmd/gen-release-notes
loadinternal: cannot find runtime/cgo
[werner@gtm-lap-024 ~/code/ecm-distro-tools/cmd/gen-release-notes]$ ldd bin/gen-release-notes
        not a dynamic executable
[werner@gtm-lap-024 ~/code/ecm-distro-tools/cmd/gen-release-notes]$  bin/gen-release-notes -h
Trace/breakpoint trap
[werner@gtm-lap-024 ~/code/ecm-distro-tools/cmd/gen-release-notes]$
```

Reply:
Is this built against the code in the fork? The binary names don't line up.

Reply:
Yes, I checked out this PR branch and ran make on it:
```
werner@gtm-lap-024:~/code/ecm-distro-tools/cmd/gen_release_notes$ make
go clean
rm -f bin/gen_release_notes
CGO_ENABLED=1 go build -tags "netgo osusergo no_stage static_build" -v -ldflags "-X main.gitSHA=7165b23553ec5b31602b93aedb17ad75f8bd17e2 -X main.version=v0.4.0 -X main.name=gen_release_notes -linkmode external -extldflags '-static -Wl,--fatal-warnings'" -o bin/gen_release_notes
github.com/rancher/ecm-distro-tools/cmd/gen_release_notes
# github.com/rancher/ecm-distro-tools/cmd/gen_release_notes
loadinternal: cannot find runtime/cgo
werner@gtm-lap-024:~/code/ecm-distro-tools/cmd/gen_release_notes$ ls bin
gen_release_notes
werner@gtm-lap-024:~/code/ecm-distro-tools/cmd/gen_release_notes$ bin/gen_release_notes
Trace/breakpoint trap
133 werner@gtm-lap-024:~/code/ecm-distro-tools/cmd/gen_release_notes$
```

Reply:
Could you build it in the container and see if the build errors are reproducible?

Reply:
Thanks for clarifying, I'll attempt to reproduce.